CRM/Contact/BAO/Contact.php

```diff
@@ -379,9 +379,6 @@ public static function &create(&$params, $fixAddress = TRUE, $invokeHooks = TRUE
       }
       else {
         $contactId = $contact->id;
-        if (isset($note['contact_id'])) {
-          $contactId = $note['contact_id'];
-        }
```

Comment:
Do you know why this was here? It looks like `$note` may or may not have been defined, depending on the loop above, but the `isset()` should protect against undefined variable notices.